kedro-telemetry/setup.py

```diff
@@ -35,6 +35,7 @@
     include_package_data=True,
     zip_safe=False,
     entry_points={
-        "kedro.cli_hooks": ["kedro-telemetry = kedro_telemetry.plugin:cli_hooks"]
+        "kedro.cli_hooks": ["kedro-telemetry = kedro_telemetry.plugin:cli_hooks"],
+        "kedro.hooks": ["kedro-telemetry = kedro_telemetry.plugin:project_telemetry_hooks"]
```

Comment:
Would it make sense to call the latter `kedro.project_hooks`? It would follow the same format as the cli hooks and also help disambiguate.